taprpc/taprootassets.proto

```diff
@@ -611,6 +615,25 @@ message ListTransfersResponse {
     repeated AssetTransfer transfers = 1;
 }
 
+message SetPendingTransferProofCourierAddrRequest {
+    // The new proof courier address to use for the target transfer output proof
+    // delivery.
+    string new_proof_courier_addr = 1;
+
+    // The hexadecimal encoded transaction ID (txid) of the anchor transaction
+    // for the target transfer.
+    string anchor_txid = 2;
```

Comment:
I think this should be the full outpoint, not just the TXID? Otherwise we aren't uniquely identifying the transfer output.

The transfer output list is per-transfer, but IIUC there is no requirement of exactly 1 transfer per TX (can I combine multiple transfers into one TX?)

So the suggested field would be `anchor_outpoint`, then in the RPC call implementation you could add a filter on `pendingParcel.Outputs` for outputs with a matching `Anchor.OutPoint`.